.github/workflows/bench.yml

```
@@ -7,13 +7,13 @@ jobs:
     steps:
 
     - name: Set up Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v2.1.3
```

We don't want to update minor versions every time. We can just fix the major version.

```suggestion
      uses: actions/setup-go@v2
```